src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+@use 'variables' as varia;
+
+.nav {
+  margin-right: 50px;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+  font-size: 12px;
+
+  &__list {
+    display: flex;
+    justify-content: flex-end;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: varia.$main-text-color;
```

Comment:
Ensure that the variable 'varia.$main-text-color' is defined in the 'variables' module. If it's not defined, this will cause a compilation error.